.eslintrc

```diff
@@ -12,13 +12,11 @@
     },
     "rules": {
         "indent": ["error", 4],
+        "max-len": ["error", 150],
```

Comment:
Why this change? The [StyleGuide](https://docs.google.com/document/d/1U8DqwOyYlNweZSa5Kn_sJW9Kx079vMclYwVi8yyPYaY/) doesn't mention this deviation from Airbnb.

Reply:
I changed this because a couple of tests were testing long strings and needed a longer `max-len`.

I've since found a much better way of running the tests.  I've undone this change and fix any linting issues.

Thanks for flagging.